HarpGuru/HarpFace/HarpCell/getDisplayValue/index.ts

```diff
@@ -11,6 +11,7 @@ type DisplayValueTuple =
 
 type Props = HarpCellProps & {
```

Comment:
Does this needs HarpCellProps in it?

Reply:
It does at the moment because it calls `analysePosition`, but I expect this will change a lot in at upcoming pull request.